tests/integration/tier1/yum-distro-sync/add-extras-repo/oracle7.yml

```diff
@@ -4,7 +4,7 @@
       yum_repository:
         name: centos7-extras
         description: CentOS extras for $basearch
-        baseurl: http://mirror.centos.org/centos-7/7/extras/$basearch/
+        baseurl: https://vault.centos.org/centos-7/7/extras/$basearch/
```

Comment:
This is for CentOS 7, shouldn't be changed

Reply:
Yeah, my bad. Thanks :D